R/count-operators.R

```diff
@@ -0,0 +1,919 @@
+                                           numeric = NA_real_),
+                  ignore.missing = TRUE,
+                  subset = NULL,
+                  warn = TRUE)
+{
+    countInputs(...,
+                operation = anyOf,
+                remove.rows = remove.rows, remove.columns = remove.columns,
+                match.elements = match.elements,
+                elements.to.count = elements.to.count,
+                ignore.missing = ignore.missing,
+                subset = subset,
+                warn = warn,
+                function.name = sQuote("AnyOf"))
+}
+
+#' @rdname CountOperators
+#' @description \code{NoneOf} checks if none of the elements satisfy some pre-specified conditions
+#'  in the case of a single input. In the case of for multiple inputs the check is performed elementwise.
+#'  It is the complement of \code{AnyOf}.
+#'
+#'  Similar to \code{\link{Sum}}, the inputs can be matched via their row and/or column names
+#'  in the case of multiple inputs with the output element being filtered by row and/or column.
+#' @export
+NoneOf <- function(...,
+                   remove.rows = NULL, remove.columns = NULL,
+                   match.elements = "Yes",
+                   elements.to.count = list(categorical = NA_character_,
+                                            numeric = NA_real_),
+                   ignore.missing =TRUE,
+                   subset = NULL,
+                   warn = TRUE)
+{
+    countInputs(...,
+                operation = noneOf,
+                remove.rows = remove.rows, remove.columns = remove.columns,
+                match.elements = match.elements,
+                elements.to.count = elements.to.count,
+                ignore.missing = ignore.missing,
+                subset = subset,
+                warn = warn,
+                function.name = sQuote("NoneOf"))
+}
+
+#' @rdname CountOperators
+#' @param x A single input to be used when performing the calculation on each row/column
+#'  dimension in \code{AnyOfEachRow}, \code{CountEachRow}, \code{NoneOfEachRow}, \code{AnyOfEachColumn}, \code{CountEachColumn}, \code{NoneOfEachColumn}.
+#' @export
+CountEachRow <- function(x,
+                         remove.rows = NULL, remove.columns = c("NET", "SUM", "Total"),
+                         elements.to.count = list(categorical = NA_integer_,
+                                                  numeric = NA_integer_),
+                         ignore.missing = TRUE,
+                         warn = TRUE)
+{
+    countEachDimension(x,
+                       dimension = 1L,
+                       operation = count,
+                       remove.rows = remove.rows,
+                       remove.columns = remove.columns,
+                       elements.to.count = elements.to.count,
+                       ignore.missing = ignore.missing,
+                       warn = warn,
+                       function.name = function.name)
```

Comment:
All the "Each" functions refer to `function.name` without defining it

Reply:
Fixed in 59a0303